Since we aren't compiling anything for the port, don't look for boost:
--- configure.orig	2015-03-23 20:37:48 UTC
+++ configure
@@ -5025,7 +5025,6 @@ else
 
     echo "Can't find boost headers. Please check the location of the boost"
     echo "distribution and rerun configure using the --with-boost=DIR option."
-    exit 1
 
 fi
 
